feedback/static/html/studio_view.html

```diff
@@ -0,0 +1,111 @@
+      may be helpful either for course improvement, or to give
+      students a chance to reflect on what they have done.</li>
+  
+    <li class="field comp-setting-entry is-set"> Prior to using this
+      block, we recommend reading about best practices in the edX
+      documentation. In particular, we do recommend asking for
+      structured, specific feedback, and we advise using this in
+      conjunction with
+      the <a href="http://edx.readthedocs.org/projects/edx-partner-course-staff/en/latest/content_experiments/content_experiments_configure.html">RCT
+      framework</a> for matrix sampling.</li>
+
+    <li class="field comp-setting-entry is-set">
+      <div class="wrapper-comp-setting">
+        <label class="label setting-label">Freeform prompt
+          <input class="input setting-input" name="freeform" id="freeform" value="{freeform}" type="text" />
+        </label>
+      </div>
+      <span class="tip setting-help">Example: Please provide us feedback on this section.</span>
+    </li>
+
+    <li class="field comp-setting-entry is-set">
+      <div class="wrapper-comp-setting">
+        <label class="label setting-label>"Freeform placeholder
+          <input class="input setting-input" name="placeholder" id="placeholder" value="{placeholder}" type="text" />
+        </label>
+      </div>
+      <span class="tip setting-help">This is shown as grayed out text before the student has answered.</span>
+    </li>
+
+    <li class="field comp-setting-entry is-set">
+      <div class="wrapper-comp-setting">
+        <label class="label setting-label">Likert icon set
+	<select name="icon_set">
+	  <option value="face">Faces (happy to sad)</option>
+	  <option value="midface">Faces (sad to happy to sad)</option>
+	  <option value="num">Numeric</option></select>
+	</select>
+	</label>
+      </div>
+      <span class="tip setting-help">We can either show happy/sad faces, or numbers 1-5.</span>
+    </li>
+
+    <li class="field comp-setting-entry is-set">
+      <div class="wrapper-comp-setting">
+        <label class="label setting-label">Likert prompt
+          <input class="input setting-input" name="likert" id="likert" value="{likert}" type="text" />
+        </label>
+      </div>
+      <span class="tip setting-help">Example: Please rate your overall experience with this section.</span>
+    </li>
+
+    <li class="field comp-setting-entry is-set">
+      <div class="wrapper-comp-setting">
+        <label class="label setting-label">Likert option 1
+          <input class="input setting-input" name="likert0" id="likert0" value="{likert0}" type="text" />
+        </label>
+      </div>
+      <span class="tip setting-help">Example: Excellent!</span>
+    </li>
+
+    <li class="field comp-setting-entry is-set">
+      <div class="wrapper-comp-setting">
+        <label class="label setting-label">Likert option 2
+          <input class="input setting-input" name="likert1" id="likert1" value="{likert1}" type="text" />
```

Comment:
add an aria-describedby="IDref" that references a unique ID on the tip/setting-help span below.